packages/nimble-blazor/NimbleBlazor/NimbleTableColumnMapping.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System.Diagnostics.CodeAnalysis;
+using Microsoft.AspNetCore.Components;
+
+namespace NimbleBlazor;
+
+public class NimbleTableColumnMapping<TKey> : NimbleTableColumn, IGroupableColumn
```

Comment:
I take it this is the analog for the `TableColumnEnumBase<T>` class in `nimble-components`. Any reason for the name change? If we like this one better, should we change the one in `nimble-components`?

Reply:
Renamed this one to `NimbleTableColumnEnumBase`.